rpc/documentation/api.md

```diff
@@ -1476,3 +1478,43 @@ The user input can be either a hexadecimal string or a mnemonic word list.
 - `InvalidArgument`: The input is invalid.
 
 **Stability:** Unstable
+
+#### `StartTicketPurchase`
+
+The `StartTicketPurchase` method enables ticket purchases for stake mining in the wallet.
+
+**Request:** `StartTicketPurchaseRequest`
+- `passphrase`: The private passphrase to unlock the wallet.
+- `account_name`: The name of the account being queried.
+- `balance_to_maintain`: The amount of funds to leave in wallet after purchasing tickets.
+- `max_fee`: The maximum ticket fee amount per KB.
+- `max_price_relative`: The scaling factor for setting the maximum ticket price, multiplied by the average price.
+- `max_price_absolute`: The maximum absolute ticket price.
+- `ticket_address`: The address to send all ticket outputs.
+- `pool_address`: The ticket pool address to send ticket fees.
+- `pool_fees`: The per ticket fee mandated by the ticket pool as a percent.
+- `max_per_block`: The maximum tickets per block. Negative number indicates one ticket every n blocks.
+
+**Response:** `StartTicketPurchaseResponse`
+
+**Expected errors:**
+
+- `FailedPrecondition`: ticket buyer is already started.
+
+**Stability:** Unstable
+
+___
+
+#### `StopTicketPurchase`
+
+The `StopTicketPurchase` method disables ticket purchases for stake mining in the wallet.
```

Comment:
same issue as the start one, you can still purchase tickets even after stopping the auto buyer.